index.js

```diff
@@ -27,6 +27,44 @@ exports.NativeScriptJsonpPlugin.prototype.apply = function (compiler) {
     });
 };
 
+exports.ExcludeUnusedElements = function () {
+    //
+}
+
+exports.ExcludeUnusedElements.prototype = {
```

Comment:
I suggest we rename this to `ExcludeUnusedElementsPlugin`. Other plugins use the same suffix.